index.js

```diff
@@ -91,6 +219,14 @@ module.exports = {
       'test:ember': 'vite build --mode test && ember test --path dist',
     };
 
+    json.scripts['test'] += ' --prefixColors auto';
+    json.scripts['lint'] += ' --prefixColors auto';
+    json.scripts['lint:fix'] += ' --prefixColors auto';
```

Comment:
this seems like a nice to have rather than something we need. I want to keep the delta between the app blueprint and the upstream blueprint as small as possible for now nad let's not add nice-to-have items 👍

Reply:
don't want want the v2 app blueprint to be as nice as possible?

Reply:
I'm not PRing these to ember-cli, fwiw, because it takes 12-18 weeks for changes to be observed without installing canary

Reply:
> don't want want the v2 app blueprint to be as nice as possible?

no, we want the v2 app blueprint **prototype** to be as minimal as possible

Reply:
https://github.com/ember-cli/ember-cli/pull/10515

Reply:
that's fine that it's upstream, but I don't want this in here too. We'll get it when we get it 👍